core/src/main/scala/org/apache/spark/api/java/JavaSparkContext.scala

```diff
@@ -530,6 +530,7 @@ class JavaSparkContext(val sc: SparkContext)
    * Create an [[org.apache.spark.Accumulator]] integer variable, which tasks can "add" values
    * to using the `add` method. Only the master can access the accumulator's `value`.
    */
+  @deprecated("use AccumulatorV2", "2.0.0")
```

Comment:
can you check if this deprecated message appear in javadoc?

I'd also update the method doc to say "Use sparkContext.longAccumulator()"